### ISSUE TITLE
False positive scenario: packages with ineffective packaging config

### ISSUE THREAD

**Issue by insin** [insin/package-config-checker#4]
Example: eslint-config-standard has the following`.npmignore`:

```
node_modules

```
1. `node_modules` is ignored by default, so `.npmignore` doesn't really do anything here
2. The npm package contains a `test/` directory, but it's given a green tick because `.npmignore` exists - having a `test/` or `tests/` directory should probably always be a negative result (and we should probably have a place to keep a list of prominent repos with module authors who disagree with that :ok_hand:)
